auth/tests.py

```diff
@@ -7,17 +7,37 @@
 from Crypto.Signature import DSS
 import jwt
 from config import AuthConfig
+from typing import Dict
+
+
+def auth_payload(user_id: str, operation_desc: Dict):
+    return json.dumps(dict(
+        user=user_id,
+        operation=operation_desc,
+    ))
+
+
+def auth_header(jwt, public_key, signature):
+    return {
+        'X-Auth-Token': jwt,
+        'X-Public-Key': public_key,
+        'X-Signature': signature,
+    }
 
 
 class AuthTest(unittest.TestCase):
 
     def create_app(self):
         return create_app_base(
-            SQLALCHEMY_DATABASE_URI='sqlite:////Users/jayden/Works/amo-storage/auth_test.db',
+            SQLALCHEMY_DATABASE_URI='sqlite://///Users/elon/Documents/develop/amo-storage/auth_test.db',
```

Comment:
Make it configurable

Reply:
This line is for overriding default configurations to run unit test.

Reply:
ok, i see.